homeassistant/components/homewizard/sensor.py

```diff
@@ -624,19 +622,21 @@ async def async_setup_entry(
 ) -> None:
     """Initialize sensors."""
 
-    data = entry.runtime_data.data.data
+    measurement: Measurement = entry.runtime_data.data.measurement
```

Comment:
I believe this type should be inferrable

Reply:
Added this for readability, but sure 👍